third_party/ruby/build-ruby.bzl

```diff
@@ -180,7 +180,12 @@ def _build_ruby_impl(ctx):
     # Build
     ctx.actions.run_shell(
         mnemonic = "BuildRuby",
-        inputs = deps + ctx.files.src + ctx.files.bundler,
+        inputs = depset(
```

Comment:
This used to (incorrectly) use a `cc` compiler outside of sandbox and thus didn't need it in sandbox